tests/LanguageServerRename/Bridge/Unit/ChainRenamerTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerRename\Tests\Bridge\Unit;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use Phpactor\Extension\LanguageServerRename\Bridge\ChainRenamer;
+use Phpactor\Extension\LanguageServerRename\Model\RenameResult;
+use Phpactor\Extension\LanguageServerRename\Model\Renamer;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\ByteOffsetRange;
+use Phpactor\TextDocument\TextDocument;
+use Phpactor\TextDocument\TextDocumentBuilder;
+use Phpactor\TextDocument\TextDocumentUri;
+use Phpactor\TextDocument\TextEdits;
+use function iterator_to_array;
+
+class ChainRenamerTest extends TestCase
+{
+    public function testGetRenameRange_OneMatch(): void
```

Comment:
For the sake of constistency I'd prefer to stick to camel case

Reply:
That's a bit of "uncle bob" thing. Sure.